modules/persistent-storage-csi-vsphere-top-aware.adoc

```diff
@@ -0,0 +1,171 @@
+// Module included in the following assemblies:
+//
+// persistent-storage-csi-vsphere.adoc
+//
+
+:content-type: PROCEDURE
+[id="persistent-storage-csi-vsphere-top-aware_{context}"]
+= vSphere Topology Awareness
+
+{product-title} provides the ability to deploy {product-title} for vSphere on different zones and regions, which allows you to deploy over multiple data centers and clusters, thus helping to avoid a single-point-of-failure.
+
+:FeatureName: vSphere topology Awareness
+include::snippets/technology-preview.adoc[leveloffset=+1]
+
+.Procedure
+To deploy {product-title} for vSphere storage on different zones and regions:
+
+. In the VMware vCenter vSphere client GUI:
+.. Create and assign zone and region type tags as needed to the hosts on the desired clusters.
+.. Create and apply "cat-zone-local-datastore" tags to the desired datastores.
+
++
+For more information about vSphere tags, see the WMware vSphere documentation.
+
+. In the `clustercsidriver` object YAML file, edit the "driverConfig" section:
+* Specify the zone and region tags that you created earlier.
+* Set "driverType" to "vSphere".
++
+[source, terminal]
+----
+~ $ oc get clustercsidriver csi.vsphere.vmware.com -o yaml
+----
++
+Example output:
++
+[source, terminal]
+----
+apiVersion: operator.openshift.io/v1
+kind: ClusterCSIDriver
+metadata:
+ name: csi.vsphere.vmware.com
+spec:
+ logLevel: Normal
+ managementState: Managed
+ observedConfig: null
+ operatorLogLevel: Normal
+ unsupportedConfigOverrides: null
+  driverConfig: 
+    driverType: vSphere <1>
+    vSphere:
+      topologyCategories: <2>
+        - k8s-zone
+        - k8s-region
+----
+<1> Ensure that "driverType" is set to "vSphere".
+<2> Zone and region tags created earlier in vCenter.
+
+. Verify that `CSINode` object has topology keys:
```

Comment:
the `CSINode` object? other examples in the docs say "the CSINode object" (without CSINode being in backticks). Might be situational.

Reply:
I checked and the file has every instance of "csinode" with backticks. They must just not be rendering as expected in the preview.